### PR TITLE
CMakeLists.txt: change version to master

### PR DESCRIPTION
**Pull request by sashko**
Signed-off-by: Oleksandr Kravchuk <oleksandr.kravchuk@pelagicore.com>

### CODE REVIEW

**Review by thenor — COMMENTED** [review]
Do we really want to do this already? Pelux 4.0 isn't released and it won't be released yet for some months.

**Comment by JEderonn** [comment]
I agree with @thenor , let's wait to update the version, at least to the major to 4.
I think it make sense to upgrade the minor more often to indicate that updates have been made, but let the  major be in sync with the PELUX release.

**Comment by sashko** [comment]
The thing is if you go to `softare-factory`, its version is shown as `3.0-alpha`, which is incorrect as it is an alpha/dev version of PELUX 4.0.

**Comment by thenor** [comment]
> The thing is if you go to `softare-factory`, its version is shown as `3.0-alpha`, which is incorrect as it is an alpha/dev version of PELUX 4.0.

I have actually been wondering about this "alpha" part. I don't think we have removed it for our releases. My issue with this change is that the update seems to indicate we are already on the 4.0 release, which we are not. We are not even in pre-release yet.

**Comment by sashko** [comment]
> > The thing is if you go to `softare-factory`, its version is shown as `3.0-alpha`, which is incorrect as it is an alpha/dev version of PELUX 4.0.
> 
> I have actually been wondering about this "alpha" part. I don't think we have removed it for our releases. My issue with this change is that the update seems to indicate we are already on the 4.0 release, which we are not. We are not even in pre-release yet.

Ture, we've forgotten to do it, which I'm trying to fix. I believe it totally makes sense to indicate it is a development version and clearly distinguish the current master from the released versions.

**Comment by thenor** [comment]
> > > The thing is if you go to `softare-factory`, its version is shown as `3.0-alpha`, which is incorrect as it is an alpha/dev version of PELUX 4.0.
> > 
> > 
> > I have actually been wondering about this "alpha" part. I don't think we have removed it for our releases. My issue with this change is that the update seems to indicate we are already on the 4.0 release, which we are not. We are not even in pre-release yet.
> 
> Ture, we've forgotten to do it, which I'm trying to fix. I believe it totally makes sense to indicate it is a development version and clearly distinguish the current master from the released versions.

In my opinion we should rather remove the alpha completely, but I will leave the decision to someone else. I don't feel that this change has high priority.

**Comment by sashko** [comment]
> > > > The thing is if you go to `softare-factory`, its version is shown as `3.0-alpha`, which is incorrect as it is an alpha/dev version of PELUX 4.0.
> > > 
> > > 
> > > I have actually been wondering about this "alpha" part. I don't think we have removed it for our releases. My issue with this change is that the update seems to indicate we are already on the 4.0 release, which we are not. We are not even in pre-release yet.
> > 
> > 
> > Ture, we've forgotten to do it, which I'm trying to fix. I believe it totally makes sense to indicate it is a development version and clearly distinguish the current master from the released versions.
> 
> In my opinion we should rather remove the alpha completely, but I will leave the decision to someone else. I don't feel that this change has high priority.

we could use "master"or whatever, but showing version as "3.0-alpha"is not right.

File: docs/conf.py.in [hunk 1]
```diff
@@ -81,8 +81,11 @@ project = u'@PROJECT_NAME@'
 copyright = u'Pelagicore AB, licensed under CC-BY-SA 4.0'
 author = u'Pelagicore AB'
 
-version = u'@PROJECT_VERSION@-alpha'
-release = u"@PROJECT_VERSION@-alpha-@REVISION@ @DATE@"
+# version should be set to @PROJECT_VERSION@ once stable PELUX version is released
+# PROJECT_VERSION is defined in CMakeLists.txt as VERSION
```
Comment (thenor) [inline docs/conf.py.in:85]: There is no such thing, as far as I know. The is a 'cmake_minimum_version' which specifies the cmake version, but the doc version is only here.

Comment (sashko) [reply docs/conf.py.in:85]: Here: https://github.com/Pelagicore/software-factory/blob/master/CMakeLists.txt

Comment (thenor) [reply docs/conf.py.in:85]: Sorry about the mistake I must have checked a CMakeLists which wasn't in the base dir.

**Review by thenor — APPROVED** [review]

**Pull request merged**